scripts/setup/setup.ps1

```diff
@@ -123,7 +127,7 @@ function Upgrade() {
 
     $installed = $false
     try {
-        .\install.ps1 -Path $adminRoot -Version $Version -SkipVerification:$SkipVerification -ServiceName $ServiceName -Port 0 -DistributablePath $DistributablePath -CertHash $CertHash
+        .\install.ps1 -Path $adminRoot -Version $Version -SkipVerification:$SkipVerification -ServiceName $ServiceName -Port 0 -DistributablePath $DistributablePath -CertHash $CertHash -LegacyConfigurations:$LegacyConfigurations
```

Comment:
This doesn't look like proper naming. It doesn't mean anything actionable. Whatever this is doing should be explicit.

Reply:
changed to `IncludeDefaultCors`